public/app/plugins/panel/geomap/GeomapPanel.tsx

```diff
@@ -44,6 +44,8 @@ interface State extends OverlayProps {
   measureMenuActive?: boolean;
 }
 
+const KEYBOARD_DESCRIPTION = `Use the arrow keys to pan the map. Use the +/- keys to zoom in and out.`;
```

Comment:
I would like some i18n for this, but since our i18n currently doesn't support plugins (although it is technically possible here, but then it is not decoupled from core)